src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -212,10 +218,14 @@ class JibriSelenium(
         // Note that the order here is important: we always want to check for no participants before we check
         // for media being received, since the call being empty will also mean Jibri is not receiving media but should
         // not cause Jibri to go unhealthy (like not receiving media when there are others in the call will).
-        val callStatusChecks = listOf(
+        val callStatusChecks = mutableListOf(
```

Comment:
Instead of using `mutableListOf`, I think you can use `buildList` which will let you embed conditional logic into the block but still keep the list immutable.

Reply:
Thank you, done!